### ISSUE TITLE
Tune Hyperparameters: learning rate, beta, lr_schedule, loss function (for more complex models), optimizer

### ISSUE THREAD

**Issue by IvanYingX** [gamladz/Furniture-StyleGAN#12]

**Comment by IvanYingX** [comment]
Added code for logging generator loss, discriminator loss, learning rate, and beta to tensorboard
